grails-app/services/org/pih/warehouse/api/GenericApiService.groovy

```diff
@@ -137,4 +141,27 @@ class GenericApiService {
         }
     }
 
+    def getGlAccountsOptions() {
```

Comment:
You should not put this here since in that way this is in no way generic. You should use the exact code that Justin suggested.